src/collection.js

```diff
@@ -127,17 +127,17 @@ function importChange(transaction, remote) {
     // Not found locally but remote change is marked as deleted; skip to
     // avoid recreation.
     if (remote.deleted) {
-      return {type: "skipped", data: remote};
+      return {type: "skipped", data: remote, previous: local};
     }
     const synced = markSynced(remote);
     transaction.create(synced);
-    return {type: "created", data: synced};
+    return {type: "created", data: synced, previous: local};
```

Comment:
Within this `if`, the `local` variable is empty. I don't think it's necessary to provide it.